src/Aurora.Infrastructure/Services/OptionsScraperCollector.cs

```diff
@@ -9,11 +9,13 @@
 using System.Reflection;
 using System.Threading.Tasks;
 
-namespace Aurora.Infrastructure
+namespace Aurora.Infrastructure.Services
 {
     public class OptionsScraperCollector : IOptionsScraperCollector
     {
         private static Dictionary<(SupportedWebsite, SearchOption), List<Type>> _scrapers = null;
+        private static List<Type> _totalScrapers = null;
```

Comment:
null is a bad practice, or at leat mark it as a nullable type List<zhopa>?

Reply:
Fixed